react/StyleGuideProvider/StyleGuideProvider.js

```diff
@@ -7,6 +7,10 @@ import classnames from 'classnames';
 import Helmet from 'react-helmet';
 import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
 
+if (typeof window !== 'undefined') {
+  require('smoothscroll-polyfill').polyfill();
```

Comment:
I guess this comes back to that broader discussion—is it the style guide's responsibility to ship a bunch of utils and polyfills? I'm not sure that sounds right.